frontend/front/src/pages/Public/Assets/theme.js

```diff
@@ -0,0 +1,59 @@
+import { createTheme } from "@mui/material";
```

Comment:
I don't think a theme right now is necessary and it might complicate the front end development for other developers. If you are comfortable with it and can show other developers how to use it then I'm ok with it.

Reply:
I am still learning how to use the createTheme function. This is a new approach in the latest version of Material-UI, utilizing variants for each MUI component (Paper, Button, Input...) . In my opinion, this could be an effective way to standardize the styles throughout the application. I will continue to learn and will provide an update within the next couple of weeks on whether or not it is a viable alternative.